### DISCUSSION TITLE
Try to build docker images using GitHub actions

### DISCUSSION THREAD

**Discussion by abelsiqueira** [asreview/asreview#1333] [Ideas]
I think it is possible to build and deploy the docker images using GitHub actions (https://github.com/marketplace/actions/build-and-push-docker-images). Looks like something like this (https://docs.docker.com/build/ci/github-actions/test-before-push/) can help test before deployment.

**Comment by J535D165** [comment]
Hi Abel, that would be great. I always forget about updating the Docker images (after Docker made the auto workflow premium). Feel free to give it a try and implement this!
